src/encoded/vis_defines.py

```diff
@@ -601,31 +604,25 @@ def lookup_colors(self):
         color = None
         altColor = None
         coloring = {}
-        biosample_term = self.dataset.get('biosample_type')
-        if biosample_term is not None:
-            if isinstance(biosample_term, list):
-                if len(biosample_term) == 1:
-                    biosample_term = biosample_term[0]
-                else:
-                    log.debug("%s has biosample_type %s that is unexpectedly a list" %
-                            (self.dataset['accession'], str(biosample_term)))
-                    biosample_term = "unknown"  # really only seen in test data!
-            coloring = BIOSAMPLE_COLOR.get(biosample_term, {})
-        if not coloring:
-            biosample_term = self.dataset.get('biosample_term_name')
-            if biosample_term is not None:
-                if isinstance(biosample_term, list):
-                    if len(biosample_term) == 1:
-                        biosample_term = biosample_term[0]
-                    else:
-                        log.debug("%s has biosample_term_name %s that is unexpectedly a list" %
-                                (self.dataset['accession'], str(biosample_term)))
-                        biosample_term = "unknown"  # really only seen in test data!
-                coloring = BIOSAMPLE_COLOR.get(biosample_term, {})
-        if not coloring:
-            organ_slims = self.dataset.get('organ_slims', [])
-            if len(organ_slims) > 1:
-                coloring = BIOSAMPLE_COLOR.get(organ_slims[1])
+        ontology = self.dataset.get('biosample_ontology')
+        term = "unknown"  # really only seen in test data!
+        if ontology is not None:
+            if not isinstance(ontology, list):
+                ontology = [ontology]
+            if len(ontology) == 1:
+                assert isinstance(ontology[0], dict)
+                term = ontology[0]['term_name']
+            else:
+                log.debug("%s has biosample_ontology %s that is unexpectedly a list" %
```

Comment:
Is this not a pylint error?  I thought logs wanted comma separated args...
```
log.debug(
    "%s has biosample_ontology %s that is unexpectedly a list",
    self.dataset['accession'],
    str([bo['@id'] for bo in ontology],
)
```
Could be wrong?